### PR TITLE
Small typo fixes (missing dashes, and double "the")

### PR DESCRIPTION
**Pull request by ajimenez2712**
'We spell our file names in lower dash case (AKA "kebab case") so we don't worry about' should read 'We spell our file names in lower dash case (AKA "kebab-case") so we don't worry about'

### CODE REVIEW

**Comment by wardbell** [comment]
Oh my my. That's a lot of the the's.  Thanks.

File: public/docs/dart/latest/tutorial/toh-pt3.jade [hunk 1]
```diff
@@ -62,7 +62,7 @@ code-example(format="." language="bash").
 
     All of our component names end in "Component".  All of our component file names end in "_component".
 
-    We spell our file names in lower dash case (AKA "kebab case") so we don't worry about
+    We spell our file names in lower dash case (AKA "kebab-case") so we don't worry about
```
Comment (kwalrath) [inline public/docs/dart/latest/tutorial/toh-pt3.jade]: Oops, this isn't actually true, so I'll need to fix it. (Dart uses lower_with_underscores, which has no cool name that I know of.)

You can either drop this file from this PR, or I'll wait for it to clear and fix the file later.

Comment (Foxandxss) [inline public/docs/dart/latest/tutorial/toh-pt3.jade]: That is snake_case. Also the PR is merged already :P

Comment (kwalrath) [inline public/docs/dart/latest/tutorial/toh-pt3.jade]: Yeah, I just saw that. :) snake_case. Cool.

**Pull request closed without merging**